src/main/scala/firrtl/ir/IR.scala

```diff
@@ -448,9 +473,16 @@ case class Module(info: Info, name: String, ports: Seq[Port], body: Statement) e
 /** External Module
   *
   * Generally used for Verilog black boxes
+  * @param defname Defined name of the external module (ie. the name Firrtl will emit)
   */
-case class ExtModule(info: Info, name: String, ports: Seq[Port]) extends DefModule {
-  def serialize: String = serializeHeader("extmodule")
+case class ExtModule(
+    info: Info,
+    name: String,
+    ports: Seq[Port],
+    defname: String,
```

Comment:
Is the reason for having defname so you can have some kind of external annotation for a module name that might vary case-to-case?